File: setup-frontend.py
#!/usr/bin/env python
import os

# Create the music-frontend directory structure
base_dir = r'C:\Users\yakic\music-frontend'
src_dir = os.path.join(base_dir, 'src')
pages_dir = os.path.join(src_dir, 'pages')

for directory in [base_dir, src_dir, pages_dir]:
    os.makedirs(directory, exist_ok=True)
    print(f"Created: {directory}")

print("Music frontend directory structure created successfully!")

File: setup-frontend.js
const fs = require('fs');
const path = require('path');

// Create the music-frontend directory structure
const baseDir = path.join('C:\\Users\\yakic\\music-frontend');
const srcDir = path.join(baseDir, 'src');
const pagesDir = path.join(srcDir, 'pages');

// Create directories
[baseDir, srcDir, pagesDir].forEach(dir => {
  if (!fs.existsSync(dir)) {
    fs.mkdirSync(dir, { recursive: true });
    console.log(`Created: ${dir}`);
  }
});

console.log('Music frontend directory structure created successfully!');
